client/lib/billing/app/views/overlays/billing.js

```diff
@@ -0,0 +1,12 @@
+import OverlayView from 'tahi/views/overlay';
+
+export default OverlayView.extend({
+  templateName: 'overlays/billing',
+  layoutName:   'layouts/overlay',
+  cardName: 'billing',
+
+  didInsertElement: function() {
+    var choice = $(".payment-method .select2-container").select2("val");
+    this.get("controller").set("selectedPaymentMethod", choice);
```

Comment:
Is there a more idiomatic ember way of doing this?

We were thinking we would add another action to the question select component (like `onLoad`), but we weren't sure.
— @DTrejo & @afomi